src/dali/plugin/input/csv/manual.py

```diff
@@ -114,6 +114,10 @@ def _load_in_file(self, transactions: List[AbstractTransaction]) -> None:
             for line in lines:
                 raw_data: str = ",".join(line).strip()
                 if not header_found:
+                    # let's let the user know they don't have enough columns
+                    if len(line) - 1 < self.__IN_NOTES_INDEX:
+                        raise ValueError("Not enough columns, the CSV must contain all columns, even optional columns.")
```

Comment:
Nice! Can you add the same logic to out and intra?

Reply:
Also let's tell users how many columns are needed:
```suggestion
                        raise ValueError(f"Not enough columns: the CSV must contain {self.__IN_NOTES_INDEX} columns.")
```